Makefile

```diff
@@ -490,11 +521,13 @@ include $(THEOS_MAKE_PATH)/library.mk
 else
 all: $(TARGET)
 $(TARGET): $(OBJECTS)
+	@echo "** BUILDING $(TARGET) FOR PLATFORM $(platform) **"
```

Comment:
OK guys, it's a small request, but please stop adding these. Please do this from now on so that I can merge these PRs without problems.

Reply:
Gone!